client/me/help/main.jsx

```diff
@@ -118,12 +118,12 @@ class Help extends PureComponent {
 						};
 
 						return (
-							<HelpResult
+							<HelpResultItem
 								key={ result.link }
 								helpLink={ result }
 								iconTypeDescription="book"
 								onClick={ trackClick }
-								localizedReadArticle={ this.props.translate( 'Read article' ) }
+								openInHelpCenter="true"
```

Comment:
+1. True is now a string. 
```suggestion
								openInHelpCenter
```